moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -12,6 +13,7 @@ import {
   useSubcomponentOptions,
   useInitialValuesOnAttributesEdit,
 } from "./utils/form";
+import * as yup from "yup";
```

Comment:
TIL `yup`. Reminds me of `zod` used for typescript aware validation. Another one is `joi`, I think. Who is naming these libraries? They are a lifesaver though!

Reply:
So true! Like there is a rule that they must be 3 letters long. `zod` sounds so cool and also reminds that I should rewatch the old Superman movies soon. 😄 
![image](https://user-images.githubusercontent.com/37249039/211839798-9adeefae-de30-4d61-9604-698279ab9274.png)

Reply:
This is the funniest comment I have read all week. So good!